app/javascript/components/layout/CustomTable.js

```diff
@@ -235,35 +235,39 @@ class CustomTable extends React.Component {
               </Col>
               <span className="ml-2 text-nowrap align-self-center">{`Displaying ${this.props.rowData.length} out of ${this.props.totalRows} rows.`}</span>
             </Row>
-          </Form>
-          {this.props.totalRows > 0 && (
-            <ReactPaginate
-              className=""
-              disableInitialCallback={true}
-              pageCount={Math.ceil(this.props.totalRows / this.props.entries)}
-              pageRangeDisplayed={4}
-              marginPagesDisplayed={1}
-              initialPage={this.props.page}
-              forcePage={this.props.page}
-              onPageChange={this.props.handlePageUpdate}
-              previousLabel="Previous"
-              nextLabel="Next"
-              breakLabel="..."
-              containerClassName="pagination mb-0"
-              activeClassName="active"
-              disabledClassName="disabled"
-              pageClassName="paginate_button page-item"
-              previousClassName="paginate_button page-item"
-              nextClassName="paginate_button page-item"
-              breakClassName="paginate_button page-item"
-              pageLinkClassName="page-link text-primary"
-              previousLinkClassName={this.props.page === 0 ? 'page-link' : 'page-link text-primary'}
-              nextLinkClassName={this.props.page === Math.ceil(this.props.totalRows / this.props.entries) - 1 ? 'page-link' : 'page-link text-primary'}
-              activeLinkClassName="page-link text-light"
-              breakLinkClassName="page-link text-primary"
-            />
-          )}
-        </div>
+          </Col>
+          <Col xs="24" className="mt-2 custom-table-footer-column-two">
```

Comment:
It should be possible to do this with just bootstrap breakpoint classes like `xs={24}` then `xl={12}` or something like that

Reply:
It is possible. The issue is that when the pagination is really large (in the middle of a really long list), the `xl` breakpoint doesnt cut it. It still overlaps the left-hand-side. With the changes in your PR, I might be able to steal some of that code. But would need your PR to be merged in beforehand